tests/integration/test_data/npm_lockfile3_yarn_registry/bom.json

```diff
@@ -0,0 +1,493 @@
+{
+  "bomFormat": "CycloneDX",
+  "components": [
+    {
+      "name": "@types/prop-types",
+      "purl": "pkg:npm/%40types/prop-types@15.7.5",
+      "type": "library",
+      "version": "15.7.5"
+    },
+    {
+      "name": "@types/react-dom",
+      "purl": "pkg:npm/%40types/react-dom@18.2.6",
+      "type": "library",
+      "version": "18.2.6"
+    },
+    {
+      "name": "@types/react",
+      "purl": "pkg:npm/%40types/react@18.2.14",
+      "type": "library",
+      "version": "18.2.14"
+    },
+    {
+      "name": "@types/scheduler",
+      "purl": "pkg:npm/%40types/scheduler@0.16.3",
+      "type": "library",
+      "version": "0.16.3"
+    },
+    {
+      "name": "abbrev",
+      "purl": "pkg:npm/abbrev@2.0.0",
+      "type": "library",
+      "version": "2.0.0"
+    },
+    {
+      "name": "accepts",
+      "purl": "pkg:npm/accepts@1.3.8",
+      "type": "library",
+      "version": "1.3.8"
+    },
+    {
+      "name": "array-flatten",
+      "purl": "pkg:npm/array-flatten@1.1.1",
+      "type": "library",
+      "version": "1.1.1"
+    },
+    {
+      "name": "bar",
+      "purl": "pkg:npm/bar@1.0.0?vcs_url=git%2Bhttps://github.com/cachito-testing/npm-cachi2-registry-yarnpkg.git%40d99077c1388bd575dc07d287b8af729da1557d1c#bar",
+      "type": "library",
+      "version": "1.0.0"
+    },
```

Comment:
Do we want the workspaces and their extra dependencies here? IMO there's value in keeping the different-lockfileVersion tests as similar as possible